devtools/run-redis.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ ! -d redis-stable/src ]; then
+    curl -O http://download.redis.io/redis-stable.tar.gz
+    tar xvzf redis-stable.tar.gz
+    rm redis-stable.tar.gz
+fi
+cd redis-stable
+make
+src/redis-server --maxmemory 1GB
```

Comment:
Ask @dgasmith about this one, but I expect that we might need more memory for some of the really big datasets like SSI. (And potentially ACCDB in the future?)